Dockerfile

```
@@ -113,7 +113,7 @@ RUN chmod +x /usr/local/bin/composer
 
 RUN apt-get update && apt-get install -y bzr && apt-get clean
 
-ENV GOLANG_VERSION 1.12
+ENV GOLANG_VERSION 1.13
```

Let's use the latest `1.13.2`

```suggestion
ENV GOLANG_VERSION 1.13.2
```

---

@ikesyo, the `1.13` label automatically refers to the latest patch version. If you explicitly use the patch version here, this line will need to be changed quite often. Is the pinning worth it?

---

> the 1.13 label automatically refers to the latest patch version.

Oh I didn't know that.